x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/i18n_texts.ts

```diff
@@ -107,6 +107,12 @@ export const i18nTexts = {
         defaultMessage: 'Maximum primary shard size',
       }
     ),
+    maxPrimaryShardDocsLabel: i18n.translate(
+      'xpack.indexLifecycleMgmt.hotPhase.maximumPrimaryShardDocsLabel',
+      {
+        defaultMessage: 'Maximum primary shard documents',
```

Comment:
Maybe we can ask for a copy review on that field label? Personally I prefer something like "Maximum documents in the primary shard"

Reply:
Plus one for changing it to "Maximum docs in the primary shard".